corehq/toggles.py

```diff
@@ -1426,6 +1426,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
 )
 
+REGEX_FIELD_VALIDATION = StaticToggle(
+    'regex_field_validation',
+    'Enable regex validation for custom data fields',
```

Comment:
super nitpick, but this is only for mobile worker user fields, right? "custom data fields" could mean a lot of things.

Reply:
actually the same UI is used for users, locations, and products